docs/AppExtensions.md

```diff
@@ -0,0 +1,27 @@
+---
+id: app-extensions
+title: App Extensions
+layout: docs
+category: Guides (iOS)
+permalink: docs/app-extensions.html
+next: native-modules-android
+previous: building-for-apple-tv
+---
+
+Building app extensions with React Native is currently not supported because of memory issues.
```

Comment:
Today Widget is not the only [app extension](https://developer.apple.com/app-extensions/) on iOS. For example Share extension is already implemented by @andrewsardone : https://github.com/andrewsardone/react-native-ios-share-extension